app/translations/cy/census_individual_gb_wls_cy.po

```diff
@@ -196,7 +195,7 @@ msgid "Include 4 or more AS levels"
 msgstr "Dylech gynnwys 4 neu fwy o gymwysterau Lefel AS (Safon UG)"
 
 msgid "Include 2 to 3 AS levels"
-msgstr "Dylech gynnwys 2-3 Lefel AS (Safon UG)"
+msgstr "Dylech gynnwys 2–3 Lefel AS (Safon UG)"
```

Comment:
Is this valid translation? @berroar is there a better way to represent 2 to 3 in Welsh?

Reply:
Rhys said it's fine

Reply:
Rhys said it's fine